src/foam/nanos/export/CSVDriver.js

```diff
@@ -22,14 +22,14 @@ foam.CLASS({
       of: 'foam.lib.csv.CSVOutputter',
       name: 'outputter',
```

Comment:
We should nuke this property and just create a new one every time you run the exportFObject method. The reason being, when you re-use the outputter, it'll be tied to a particular type of FObject and it will not re-output the columns.

Reply:
Done.